libyul/backends/evm/EVMDialect.cpp

```diff
@@ -82,7 +82,7 @@ pair<YulString, BuiltinFunctionForEVM> createEVMFunction(
 		_assembly.appendInstruction(_instruction);
 	};
 
-	return {f.name, move(f)};
+	return {f.name, std::move(f)};
```

Comment:
This looks like it depends on the order of evaluation of arguments, which is undefined in C++.

Reply:
https://github.com/ethereum/solidity/pull/13449